README

```diff
@@ -414,6 +414,15 @@ INSTALLATION
 
     By default INSTALLDIRS is set to site.
 
+DOCKER USAGE
+    In order to run pgbadger in docker container, you have to build it:
+
+            docker build . -t dalibo/pgbadger
+
+    After you can run with:
+
+            docker run dalibo/pgbadger --help
```

Comment:
What do you think of suggesting volumes for logs and report ?

Reply:
I can complete documentation with examples but I'll base on `docker run` documentation

Reply:
Yes, just something like

``` console
$ docker run --volume /var/log/postgresql:/logs --volume ${PWD}/outdir:/outdir dalibo/pgbadger -O /outdir ...
```